packages/opentelemetry-node/src/spanprocessor.ts

```diff
@@ -50,17 +50,6 @@ export class SentrySpanProcessor implements OtelSpanProcessor {
    * @inheritDoc
    */
   public onStart(otelSpan: OtelSpan, parentContext: Context): void {
-    const hub = getCurrentHub();
-    if (!hub) {
-      __DEBUG_BUILD__ && logger.error('SentrySpanProcessor has triggered onStart before a hub has been setup.');
-      return;
-    }
-    const scope = hub.getScope();
-    if (!scope) {
-      __DEBUG_BUILD__ && logger.error('SentrySpanProcessor has triggered onStart before a scope has been setup.');
```

Comment:
We can remove all of this because a hub should always be defined (and scope is actually never used anywhere).

Reply:
nice! 🔥